src/components/Quill/Quill.tsx

```diff
@@ -9,6 +9,7 @@ import { generateWebViewIndex } from './resources/generateWebViewIndex';
 interface IProps {
   content?: DeltaStatic;
   onContentChange: (content: DeltaStatic) => any;
+  containerStyle: ViewStyle | undefined;
```

Comment:
```suggestion
  containerStyle?: ViewStyle;
```

Dit is de TypeScript syntax voor optional